jekyll/code_carrousel.html

```diff
@@ -175,31 +178,44 @@
 
 for word in tokenize("This is an example"):
   echo word
+
+let (ok, year, month, day) = scanTuple("1000-01-01", "$i-$i-$i")
+if ok:
+  assert year == 1000
+  assert month == 1
+  assert day == 1
 {% endhighlight %}
 </code>
 
 
 <code id="sample4" class="sample">
 {% highlight nim %}
-import sugar, tables, sets
+import std/[sugar, tables, sets, sequtils, strutils]
 
-let variable0 = collect(newSeq):
+let variable0 = collect:
```

Comment:
Isn't it a feature of 'collect' of devel branch? The improvement should be removed or deferred.

Reply:
Theres no clear guideline on it, I used devel, 
because according to the latest poll most users use devel (or nigthly).

But I can use the old anyway...